app/assets/javascripts/components/authentication/login_form.es6.jsx

```diff
@@ -46,6 +46,8 @@ class LoginForm extends Component {
     email.addEventListener('input', this.generateHandler('email'));
     var password = ReactDOM.findDOMNode(this.refs.password);
     password.addEventListener('input', this.generateHandler('password'));
+    var node = ReactDOM.findDOMNode(this.refs.container);
```

Comment:
Let's call this `container` and move it to the top since there are a lot of other `nodes`.